core/test/qa_Settings.cpp

```diff
@@ -131,8 +123,8 @@ const boost::ut::suite SettingsTests = [] {
         expect(eq(src.sample_rate, 42.f)) << "check map constructor";
         expect(eq(src._nSamplesProduced, gr::Size_t(0))) << "default value";
         expect(eq(src.settings().getNAutoUpdateParameters(), 1UZ));
-        expect(eq(src.settings().autoUpdateParameters().size(), 3UL));  // 3 base + 0 derived
-        expect(eq(src.settings().autoForwardParameters().size(), 1UL)); // sample_rate
+        expect(eq(src.settings().autoUpdateParameters().size(), 3UL));   // 3 base + 0 derived
+        expect(eq(src.settings().autoForwardParameters().size(), 16UL)); // all gr::tag::kDefaultTags
```

Comment:
Using `gr::tag::kDefaultTags.size()` would be more expressive, avoid the code comment, and simplify later refactoring.